Documentation/UsingSetting/AddTypoScriptWithExtensions.rst

```diff
@@ -6,60 +6,167 @@
 Add TypoScript in your extension
 ================================
 
+..  versionchanged:: 13.1
+    TypoScript on a per site basis can now be included via
+    :ref:`sites and sets <typoscript-site-sets>`.
+
 .. note::
 
    This part is written for extension developers.
 
 
 .. index:: TypoScript in extensions; File locations
 
+.. _extdev-add-typoscript-extension:
+
 Create TypoScript files in your extension
 =========================================
 
-TypoScript files should have the ending :file:`.typoscript`.
+TypoScript files **must** have the ending :file:`.typoscript`.
 
-They are located in :file:`Configuration/TypoScript` within your
-extension.
+They are located in :path:`Configuration/Sets/MySet` within your
```

Comment:
Hhm but having TypoScript outside the "sets" scope is still possible; isn't the section below meant for the new placement syntax, and we shouldn't change the old?

Reply:
Yup, the old location is still possible. The site sets is a new way when using - well - site sets. You can still go the old way without site sets.

Reply:
Starting with TYPO3 v13 not using site sets is not recommended anymore. You can still put your files to be included in other directions

Reply:
Then this should be mentioned. Where is the source for the recommendation? How would you configure TypoScript for all sites in your installation?